src/gsi-ncdiag/test_gsidiag.py.in

```diff
@@ -20,10 +20,22 @@ parser.add_argument('-t', '--type',
 parser.add_argument('-p', '--platform',
                     help="platform to run, only works on conv",
                     type=str)
+parser.add_argument('-obsbias', '--add_obsbias',
```

Comment:
Same comment/suggestions as before. The - args shouldn't be that long if possible.

Reply:
@CoryMartin-NOAA Will do!!